### DISCUSSION TITLE
Regarding the infinite table and the question about the select-all checkbox.

### DISCUSSION THREAD

**Discussion by 4blacktea** [nocodb/nocodb#9953] [General]
I've noticed that Nocodb has recently released the feature for infinite tables, but I've encountered an issue where the checkbox for selecting all items is no longer displayed. I'm wondering if there's a way to configure it to switch to a paginated mode, or if there are alternative methods to achieve the functionality of selecting all items.

**Comment by o1lab** [comment]
We heard you! Fix will be available this week. Thank you

**Reply by 4blacktea** [reply]
Great, thank you for your reply and strong support.
